pages/agent/v3.md

```diff
@@ -153,38 +147,39 @@ The API is exposed using a Unix Domain Socket. Unlike the `job-api`, the path to
 
 ## Promoted experiments
 
-We've recently promoted some experiments into full features.
+The following features started as experiments before being promoted to fully supported features.
```

Comment:
Avoids being tied to a particular time, then lists features from newest to oldest.